### DISCUSSION TITLE
iCasa Pulse 8S: No messages

### DISCUSSION THREAD

**Discussion by alpha23** [Koenkk/zigbee2mqtt#12970] [Devices]
Hi,

I´ve added an iCasa Pulse 8S switch. Pairing and  interview worked fine but i get no MQTT messages when I press a button. All I can do is to add a binding between a lamp or a group and the OnOff cluster.

Is it possible to send messages with this device?

Best regards
Mark Herzberg

**Comment by Ryoga85** [comment]
Is there any solution for this? I'm having the same issue. I see no events from the buttons, all they do is send OnOff commands to a random group number.

**Reply by Ryoga85** [reply]
Never mind, restarting Z2M solved the problem for me.
